src/scripts/federal-holidays-reminder.js

```diff
@@ -0,0 +1,88 @@
+const moment = require("moment-timezone");
+const scheduler = require("node-schedule");
+const holidays = require("@18f/us-federal-holidays");
+const {
+  slack: { postMessage },
+} = require("../utils");
+
+const CHANNEL = process.env.HOLIDAY_REMINDER_CHANNEL || "general";
+const TIMEZONE = process.env.HOLIDAY_REMINDER_TIMEZONE || "America/New_York";
+const reportingTime = moment(
+  process.env.HOLIDAY_REMINDER_TIME || "15:00",
+  "HH:mm"
+);
+const SUPPRESS_HERE = process.env.HOLIDAY_REMINDER_SUPPRESS_HERE;
+
+const suppressHere = (() => {
+  if (SUPPRESS_HERE) {
+    const upper = SUPPRESS_HERE.toUpperCase();
+    if (upper === "TRUE" || upper === "YES" || upper === "Y") {
```

Comment:
I am ashamed. There are libraries that will do truthy-type checks for strings and I should just use one of those. 😬  I will do that. OR, I may just take out the `SUPPRESS_HERE` altogether, since we don't actually use it at all. It was a premature optimization to ease some concerns that never actually materialized.